packages/vats/src/proposals/testUpgradedBoard.js

```diff
@@ -0,0 +1,35 @@
+import { E, Far } from '@endo/far';
+import { Stable } from '@agoric/internal/src/tokens.js';
+
+export const testUpgradedBoard = async ({
+  consume: { board },
+  brand: {
+    consume: { [Stable.symbol]: stableBrandP },
+  },
+}) => {
+  // /////// can we store something and get it back?  ////////
+  const thing1 = Far('thing1', {});
+  const thing1Id = await E(board).getId(thing1);
+  assert(thing1Id.match(/^board0[0-9]+$/));
+
+  // /////// can we retrieve something stored long ago?  ////////
```

Comment:
what is the thing stored long ago that we're retrieving?

Reply:
The brand. The comment was displaced from its subject.